src/corelibs/U2Script/src/UgeneContextWrapper.cpp

```diff
@@ -105,6 +111,13 @@ static void setDataSearchPaths() {
     } else if (QDir(AppContext::getWorkingDirectoryPath() + relativeDevDataDir).exists()) {
         coreLog.info("Added path: " + AppContext::getWorkingDirectoryPath() + relativeDevDataDir);
         dataSearchPaths.push_back(AppContext::getWorkingDirectoryPath() + relativeDevDataDir);
+#ifdef Q_OS_DARWIN
+    } else {
+        QString dir = BundleInfoMac::getDataSearchPath();
+        if (QDir(dir).exists()) {
```

Comment:
You should compare with an empty string here: the getDataSearchPath already checks if the dir exists.

Reply:
done